### DISCUSSION TITLE
How to fire a task after publish without open or load the project

### DISCUSSION THREAD

**Discussion by Jazayery** [quartznet/quartznet#1556] [Q&A] [answered]
Hi after publish on IIS (Upload on the Internet) if you didn't open page and refresh page the task never fire.
do you have any idea?

I try to find any solution but I didn't find, If you know please send a link.

Thanks.

**Comment by lahma** [comment, marked as answer]
Well here's a link: https://www.quartz-scheduler.net/documentation/faq.html#scheduler-keeps-stopping-when-application-pool-gets-recycled

IIS just isn't ideal for background jobs.

**Reply by Jazayery** [reply]
Thank you for your attention, 
I do it but when the task time set 5 min only 4 times task fired.
